lib/byte_safe_strings.php

```diff
@@ -50,8 +50,10 @@ function RandomCompat_strlen($binary_string)
                     'RandomCompat_strlen() expects a string'
                 );
             }
+
             return mb_strlen($binary_string, '8bit');
         }
+
```

Comment:
> Best option: Contributors don't alter code style to match their own preferences.

StyleCI bipasses this if autosending and merging is enabled. It'll revert any bad cs changes immidately after PR merging.